index.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
-module.exports = function (str) {
-	var isExtendedLengthPath = /^\\\\\?\\/.test(str);
-	var hasNonAscii = /[^\x00-\x80]+/.test(str);
+module.exports = str => {
+	const isExtendedLengthPath = /^\\\\\?\\/.test(str);
+	const hasNonAscii = /[^\x00-\x80]+/.test(str); // eslint-disable-line no-control-regex
```

Comment:
Can you switch those to Unicode escapes? https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-hex-escape.md